ds000117/sub-01/ses-meg/meg/sub-01_ses-meg_task-facerecognition_run-01_events.tsv

```diff
@@ -1,147 +1,147 @@
-314.687	0	315237	Famous	5	meg/f021.bmp
-318.05	0	318600	Scrambled	17	meg/s052.bmp
-321.633	0	322183	Scrambled	18	meg/s052.bmp
-325.087	0	325637	Scrambled	17	meg/s055.bmp
-328.487	0	329037	Unfamiliar	15	meg/u086.bmp
-332.125	0	332675	Scrambled	17	meg/s123.bmp
-335.598	0	336148	Scrambled	18	meg/s123.bmp
-339.089	0	339639	Famous	5	meg/f013.bmp
-342.709	0	343259	Famous	6	meg/f013.bmp
-346.293	0	346843	Famous	5	meg/f026.bmp
-349.821	0	350371	Famous	7	meg/f021.bmp
-353.331	0	353881	Famous	5	meg/f128.bmp
-356.84	0	357390	Famous	6	meg/f128.bmp
-360.332	0	360882	Scrambled	17	meg/s102.bmp
-364.007	0	364557	Scrambled	19	meg/s055.bmp
-367.498	0	368048	Scrambled	17	meg/s136.bmp
-370.898	0	371448	Scrambled	18	meg/s136.bmp
-374.536	0	375086	Unfamiliar	13	meg/u134.bmp
-378.028	0	378578	Scrambled	17	meg/s010.bmp
-381.574	0	382124	Famous	7	meg/f026.bmp
-384.901	0	385451	Scrambled	17	meg/s084.bmp
-388.41	0	388960	Famous	5	meg/f009.bmp
-391.846	0	392396	Famous	6	meg/f009.bmp
-395.411	0	395961	Scrambled	19	meg/s102.bmp
-399.068	0	399618	Famous	5	meg/f129.bmp
-402.504	0	403054	Unfamiliar	13	meg/u051.bmp
-406.014	0	406564	Unfamiliar	15	meg/u134.bmp
-409.487	0	410037	Unfamiliar	13	meg/u140.bmp
-412.776	0	413326	Scrambled	19	meg/s010.bmp
-416.157	0	416707	Famous	5	meg/f096.bmp
-419.648	0	420198	Famous	6	meg/f096.bmp
-423.067	0	423617	Scrambled	19	meg/s084.bmp
-426.447	0	426997	Unfamiliar	13	meg/u139.bmp
-430.012	0	430562	Unfamiliar	14	meg/u139.bmp
-433.375	0	433925	Scrambled	17	meg/s140.bmp
-436.811	0	437361	Scrambled	18	meg/s140.bmp
-440.303	0	440853	Famous	7	meg/f129.bmp
-443.849	0	444399	Famous	5	meg/f049.bmp
-447.304	0	447854	Famous	6	meg/f049.bmp
-450.869	0	451419	Unfamiliar	15	meg/u051.bmp
-454.36	0	454910	Famous	5	meg/f140.bmp
-457.668	0	458218	Unfamiliar	15	meg/u140.bmp
-461.233	0	461783	Unfamiliar	13	meg/u017.bmp
-464.761	0	465311	Unfamiliar	14	meg/u017.bmp
-468.105	0	468655	Famous	5	meg/f033.bmp
-471.524	0	472074	Famous	6	meg/f033.bmp
-475.106	0	475656	Famous	5	meg/f060.bmp
-478.487	0	479037	Famous	6	meg/f060.bmp
-481.997	0	482547	Scrambled	17	meg/s098.bmp
-485.397	0	485947	Scrambled	18	meg/s098.bmp
-488.87	0	489420	Scrambled	17	meg/s110.bmp
-492.398	0	492948	Scrambled	18	meg/s110.bmp
-495.999	0	496549	Famous	7	meg/f140.bmp
-499.436	0	499986	Unfamiliar	13	meg/u073.bmp
-502.835	0	503385	Famous	5	meg/f137.bmp
-506.271	0	506821	Unfamiliar	13	meg/u066.bmp
-509.744	0	510294	Unfamiliar	13	meg/u078.bmp
-513.254	0	513804	Unfamiliar	14	meg/u078.bmp
-516.562	0	517112	Famous	5	meg/f131.bmp
-520.09	0	520640	Unfamiliar	13	meg/u105.bmp
-523.526	0	524076	Unfamiliar	15	meg/u073.bmp
-527.073	0	527623	Unfamiliar	13	meg/u132.bmp
-530.601	0	531151	Famous	7	meg/f137.bmp
+24.2073	0	26628	Unfamiliar	13	meg/u032.bmp
+27.2473	0	29972	Unfamiliar	14	meg/u032.bmp
+30.3545	0	33390	Unfamiliar	13	meg/u088.bmp
+33.3618	0	36698	Unfamiliar	13	meg/u084.bmp
+36.5536	0	40209	Famous	5	meg/f123.bmp
+39.5764	0	43534	Unfamiliar	13	meg/u022.bmp
+42.8	0	47080	Famous	5	meg/f094.bmp
```

Comment:
are you sure these onsets are correct? Can you share the scripts you used to compute them? Thanks

Reply:
Hmm, I don't have the scripts, but basically each onset is equal to [(Rik's onset) + 550]/1100. For sub-01 run-01, here is Rik's file: ftp://ftp.mrc-cbu.cam.ac.uk/personal/rik.henson/wakemandg_hensonrn/Sub01/MEEG/Trials/run_01_trldef.txt

So for line 1, Rik's onset was 26078. Thus [(26078) + 550]/1100 = 24.2073, which is the onset on line 1 of my file. Does that sound right? How do yours differ?

Reply:
hmm ... I don't know how these were found. What is 550 here? Did you look at the trigger channels? Here it is for the first file:

![trigger_channel](https://user-images.githubusercontent.com/15852194/29944249-656f03c8-8eba-11e7-9254-373442f01dfe.png)

Don't you think the onset is less than 24 s?

Reply:
Yes, it does look like its less than 24. I'm not actually sure what 550 means but this is how Vladimir and Rik instructed me to calculate them. They're not on github AFAIK but I'll forward them this discussion so we can discuss over email.